_maps/shuttles/whiteship_cere.dmm

```diff
@@ -3,625 +3,1400 @@
+/obj/machinery/airalarm/all_access{
+	dir = 4;
+	pixel_x = 24
+	},
+/turf/open/floor/catwalk_floor/iron_dark,
+/area/shuttle/abandoned/cargo)
+"tg" = (
+/obj/machinery/door/poddoor{
+	id = "cerewhiteright"
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 4
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/shuttle/abandoned/cargo)
+"tE" = (
+/obj/machinery/power/port_gen/pacman/pre_loaded,
+/obj/structure/cable,
+/turf/open/floor/plating,
+/area/shuttle/abandoned/engine)
+"ug" = (
+/obj/machinery/suit_storage_unit/spaceruin,
+/turf/open/floor/plating,
+/area/shuttle/abandoned/engine)
+"ux" = (
+/obj/structure/closet/secure_closet/freezer/kitchen/maintenance,
+/obj/effect/spawner/random/food_or_drink/donkpockets,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/firealarm/directional/west,
+/turf/open/floor/mineral/titanium,
+/area/shuttle/abandoned/bar)
+"vb" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/firealarm/directional/south,
+/turf/open/floor/pod/dark,
+/area/shuttle/abandoned/cargo)
+"vm" = (
+/obj/structure/table/reinforced,
+/obj/effect/spawner/random/food_or_drink/donkpockets,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/airalarm/all_access{
+	dir = 4;
+	pixel_x = 24
+	},
+/turf/open/floor/mineral/titanium,
+/area/shuttle/abandoned/bar)
+"wV" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/mob/living/basic/migo,
+/turf/open/floor/pod/dark,
+/area/shuttle/abandoned/cargo)
+"xA" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/atmospherics/components/unary/vent_pump/on{
+	dir = 4
+	},
+/turf/open/floor/pod/dark,
+/area/shuttle/abandoned/cargo)
+"yp" = (
+/obj/machinery/vending/boozeomat/all_access,
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/wood,
+/area/shuttle/abandoned/bar)
+"Ae" = (
+/obj/machinery/power/apc/auto_name/directional/north,
+/obj/structure/cable,
+/obj/structure/closet/crate{
+	name = "spare equipment crate"
+	},
+/obj/item/storage/toolbox/mechanical,
+/obj/item/storage/toolbox/electrical,
+/turf/open/floor/plating,
+/area/shuttle/abandoned/engine)
+"Ai" = (
+/obj/structure/cable,
+/obj/structure/cable/layer1,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/mob_spawn/corpse/human/miner{
+	brute_damage = 200
+	},
+/obj/item/knife/combat/survival,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/machinery/firealarm/directional/east,
+/obj/effect/decal/cleanable/blood/gibs/old,
+/turf/open/floor/wood,
+/area/shuttle/abandoned/crew)
+"Bv" = (
+/obj/effect/decal/remains/human,
+/obj/item/clothing/head/hats/caphat/parade{
+	name = "old captain's cap";
+	desc = "A damaged Captain's cap, smelling faintly of blood.";
+	pixel_y = 4
+	},
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/item/gun/energy/laser/retro,
+/obj/item/paper/crumpled/bloody{
+	default_raw_text = "Those... Things got on. My crew are dead, I have bolted myself in. I hear slithering outside, and I am starving. Goodbye."
+	},
```

Comment:
I always love included lore, I would consider making the papers specifically code-side subtypes instead of mapped in variants so you can put more love into the writing without it being a pain.

Reply:
i could probably do that, one moment

Reply:
my work is done